experimental/waveshare_eink/linux_kernel_modules/mangOH_yellow_ws213.c

```diff
@@ -0,0 +1,119 @@
+/* This files provides configuration data  & SPI setup for the
+ * Waveshare 213 for the mangOH Yellow. Still think TODO one file
+ * for both Red & Yellow boards
+ */
+
+#include <linux/module.h>
+#include <linux/spi/spi.h>
+#include <linux/gpio/driver.h>
+
+#if defined(CONFIG_ARCH_MSM9615) /* For WPX5XX */
+#include <../arch/arm/mach-msm/board-9615.h>
+#elif defined(CONFIG_ARCH_MDM9607) /* For WP76XX */
+#include <../arch/arm/mach-msm/include/mach/swimcu.h>
+#endif
+
+#include "fb_waveshare_eink.h"
+
+#define SPI_BUS		32766
+#define SPI_BUS_CS1	1
+/*
+ * The minimum clock cycle duration supported by the waveshare 2.13 Inch display
+ * is 250ns which translates to 4MHz bus speed
+ */
+#define SPI_BUS_SPEED	4000000
+
+const char eink_device_name[] = "waveshare_213";
+static struct spi_device *eink_device;
+
+/* On the Yellow the busy gpio is off the IO expander unlike the Red */
+/* TODO: Test the settings for the WP85 on a mangOH Yellow */
+#if defined(CONFIG_ARCH_MSM9615)
+#define CF3_GPIO35	SWIMCU_GPIO_TO_SYS(1)
+#define CF3_GPIO40	SWIMCU_GPIO_TO_SYS(6)
+#define GPIO_OFFSET	1
+#elif defined(CONFIG_ARCH_MDM9607)
+#define CF3_GPIO35	 (37)
+#define CF3_GPIO40	SWIMCU_GPIO_TO_SYS(3)
+#define GPIO_OFFSET	1
+#endif
+
+static struct waveshare_eink_platform_data ws213_pdata = {
+	.rst_gpio	= CF3_GPIO35,
+	.dc_gpio	= CF3_GPIO40,
+	.busy_gpio	= -1,
+};
+
+/* We should patch the kernel in meta-mangoh to export this routine */
```

Comment:
I disagree with patching the kernel to expose static functions just because they're useful.  I think it's better to just define this function here as you have done already.